app/models/zaikio/access_token.rb

```diff
@@ -18,7 +18,7 @@ def self.build_from_access_token(access_token, requested_scopes: nil)
         bearer_id: access_token.params["bearer"]["id"],
         audience: access_token.params["audiences"].first,
         token: access_token.token,
-        refresh_token: access_token.refresh_token,
+        refresh_token: (access_token.refresh_token unless skip_refresh_token),
```

Comment:
`unless ... skip` is a double negative, can we flip it?

```ruby
def self.build_from_access_token(access_token, requested_scopes: nil, include_refresh_token: true)
```

```suggestion
        refresh_token: (access_token.refresh_token if include_refresh_token),
```